.github/workflows/build-test.yml

```diff
@@ -164,7 +164,12 @@ jobs:
     - name: Show the selected Rust toolchain
       run: rustup show
 
-    # Job specific dependencies
+    # Job-specific dependencies
+    - name: Attempt to load download cache
+      uses: actions/cache@v3
+      with:
+        key: download-cache
+        path: /tmp/icu4x
```

Comment:
Nit: This is an unrelated change that should be in its own PR

Reply:
It's not. This PR builds testdata from a tag instead of the locally downloaded subset of sources.